apps/deploy-web/src/components/deployments/DeploymentDepositModal.tsx

```diff
@@ -259,6 +295,17 @@ export const DeploymentDepositModal: React.FunctionComponent<DeploymentDepositMo
                 );
               }}
             />
+            <div className="mt-1 flex justify-end text-xs">
+              {user?.userId ? (
+                <LinkTo onClick={() => goToCheckout()} className="text-primary">
```

Comment:
blocking: only users which are logged in and verified can use the checkout and top up. we already have utilities for this. e.g. here in [YourAccount](https://github.com/akash-network/console/blob/main/apps/deploy-web/src/components/home/YourAccount.tsx#L235)

Reply:
if done like so the link to sign might not be needed

Reply:
another concern: is this visible to all the users? should only be for managed wallet users